src/elarm_log.erl

```diff
@@ -73,7 +73,8 @@ add_comment(_Alarm, _Comment, State) ->
     {ok, State}.
 
 %% Automatically clear an alarm
--spec clear(alarm(), clear_reason(), #log_state{}) -> {ok, #log_state{}} | {error, term()}.
+-spec clear(alarm(), clear_reason(), #log_state{}) -> {ok, #log_state{}} |
```

Comment:
The spec in line 76 uses a record, please define a type for the record and use that instead.